serviceAccessConfig/lib/serviceAccessConfig/apache.py

```diff
@@ -45,7 +45,9 @@ def _get_apache_ip_directive(self):
         """Figure out the version of Apache and return the appropriate
            Apache directive for IP address filtering"""
         apachectl = None
+        closing_directive = None
```

Comment:
Consider using an empty string, instead of `None` to avoid type casting issues (e.g. `TypeError: cannot concatenate 'str' and 'NoneType' objects`)